### ISSUE TITLE
OrientGraph.dropIndex() doesn't completely delete index in Gremlin?

### ISSUE THREAD

**Issue by lebedov** [orientechnologies/orientdb#4730]
Say I create a new database in the OrientDB console using something like
```
orientdb {db=foo}> create database remote:localhost/foo admin admin memory graph
```
and then create an index in the database via the Gremlin console as follows:
```
gremlin> g = OrientGraph("remote:localhost/foo");
gremlin> g.createIndex("ind", Vertex.class);
==>index[ind:Vertex]
```
If I subsequently delete the index with `g.dropIndex("ind")`, `g.getIndices()` seems to indicate that the index is gone:
```
gremlin> g.dropIndex("ind")
==>null
gremlin> g.getIndices()
gremlin>
```
However, if I try to recreate the index by calling `g.createIndex("ind", Vertex.class)`, I observe the following error message:
```
Index with name __@recordmap@___ind already exists.
```
The ` __@recordmap@___ind` index also appears if I subsequently run the `list indexes` command from the OrientDB Console. If I explicitly run `g.dropIndex("__@recordmap@___ind")`, the index vanishes from the output of `list indexes`. This suggests that `g.dropIndex()` might not be handling its arguments properly.

I'm using OrientDB 2.0.13.

**Comment by Pressy** [comment]
The same issue for me. 
I couldn't run my application because of this issue. I use rexster server to access the database. Whenever rexster initializes the database while starting the application, this error is reported.

I have also tried deleting the index manually but is of no use. It again reports the same error.

I am using OrientDB 2.1.5  and Rexster 2.6.0 and bulbs in python.

Any way to solve this issue is welcomed.

**Comment by luigidellaquila** [comment]
Hi guys,

This issue was fixed a few months ago in 2.2

Closing

Thanks

Luigi
